src/index.html

```diff
@@ -8,9 +8,36 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
```

Comment:
Fix the syntax according to the checklist, please 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/3b4e3f08-899d-4c6b-851a-5e45ac257e10)
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/05352dc5-c87f-4f42-ad1e-7a9443ba96e2)